packages/react-components/react-combobox/src/components/Option/Option.test.tsx

```diff
@@ -36,6 +37,15 @@ describe('Option', () => {
     expect(result.container).toMatchSnapshot();
   });
 
+  it('renders child content', () => {
+    const result = render(
+      <Option text="Default">
+        <b>Text content</b>
+      </Option>,
+    );
+    expect(result.container).toMatchSnapshot();
```

Comment:
Instead of a snapshot test, it'd be ideal to use `getByText`/`queryByText` to check for the existence of the child content. E.g.
```suggestion
    expect(result.getByText('Text content').tagName).toBe('b');
    expect(result.queryByText('Default')).toBeNull();
```